app/agents/voice/automatic/services/mcp/utils.py

```
@@ -0,0 +1,72 @@
+                    if (
+                        chart_data
+                        and isinstance(chart_data, dict)
+                        and chart_data.get("uiComponent")
+                    ):
+                        try:
+                            session_id = get_current_session_id()
+                            if session_id:
+                                _register_pending_chart_emission(session_id, chart_data)
+                        except Exception:
+                            # Don't break MCP flow if chart registration fails
+                            pass
```

Ruff (0.14.2)

`try`-`except`-`pass` detected, consider logging the exception

(S110)